calico-enterprise/networking/egress/egress-gateway-azure.mdx

```diff
@@ -137,7 +137,6 @@ Azure Route Server deployment is per VNet, as such each VNet with egress gateway
 - [Verify the feature operation](#verify-the-feature-operation)
 - [Controlling the use of egress gateways](#controlling-the-use-of-egress-gateways)
 - [Policy enforcement for flows via an egress gateway](#policy-enforcement-for-flows-via-an-egress-gateway)
-- [Upgrading egress gateways](#upgrading-egress-gateways)
```

Comment:
List should use active verbs:
Verify
Control
Enforce policy for flows...
Upgrade egress gateways

Reply:
FIxed this in the aws and on-prem docs. upgrade is not applicable for azure as this is the first release.